dask/dataframe/io/parquet/core.py

```diff
@@ -924,4 +948,35 @@ def aggregate_row_groups(parts, stats, chunksize):
     return parts_agg, stats_agg
 
 
+def _write_partition(
+    write_func,
+    df,
+    path,
+    fs,
+    filename,
+    partition_on,
+    return_metadata,
+    kwargs,
+):
+    """Engine.write_partition wrapper.
+    The only purpose of this wrapper is to unpack kwargs.
```

Comment:
In these cases it is common to use the `apply` function

```python
(apply, func, args, kwargs)
```

Reply:
Ah - Thanks!  I completely forgot about apply.  That is perfect.